atomate/vasp/fireworks/core.py

```diff
@@ -284,6 +324,89 @@ def __init__(self, structure, name="static dielectric", vasp_cmd="vasp",
             structure.composition.reduced_formula, name), **kwargs)
 
 
+class LcalcpolFW(Firework):
+    def __init__(self, structure, name="static dipole moment", static_name="static", vasp_cmd="vasp",
+                 vasp_input_set=None, vasp_input_set_params=None, db_file=None, parents=None,
+                 gap_threshold=0.010, interpolate=False, start=None, end=None, this_image=0, nimages=5, **kwargs):
+        """
+        Standard static calculation Firework for dipole moment. The calculation will not calculate the polarization
+        if the band gap of the SCF calculation is metallic (have a band gap less than the gap_threshold).
+
+        The SCF calculation can be provided as a previous run or can be computed within this Firework.
+
+        Args:
+            structure (Structure): Input structure. For an interpolation, this is a dummy structure.
+            name (str): Name for the polarization FireWork.
+            static_name (str): Name for the SCF run to be used in PassCalcLoc if copy_vasp_outputs != True.
+            vasp_cmd (str): Command to run vasp.
+            copy_vasp_outputs (bool): Whether to copy outputs from previous run. Defaults to False.
+            vasp_input_set (str): string name for the VASP input set (e.g., "MITMDVaspInputSet").
+            db_file (str): Path to file specifying db credentials.
+            parents (Firework): Parents of this particular Firework. FW or list of FWS.
+            calc_loc (str or True): Name of the previous SCF calculation to be use for CopyVaspOutputs.
+                True defaults to parent.
+            gap_threshold: Band gap cutoff for determining whether polarization calculation will proceed from
+                SCF band gap.
+            defuse_children (bool): defuse children FireWorks if StaticFW shows material is metallic.
+            exit_firework (bool): do not run polarization calculation if StaticFW shows material is metallic.
```

Comment:
"exit_if_metallic"

Reply:
actually this doesn't even seem to be an arg anymore?

Reply:
Done.